ENGAGEHF/Dashboard/ToDo/Views/ToDoRows.swift

```diff
@@ -0,0 +1,48 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct ToDoRow: View {
+    @Binding var task: ToDoItem
+    @State var taskComplete = false
+    
+    var body: some View {
+        HStack {
+            Text(task.task)
+            Spacer()
+            Button {
+                taskComplete.toggle()
+            } label: {
+                Image(systemName: taskComplete ? "checkmark.square" : "square")
```

Comment:
I think the task completion label could be a bit larger. You can use `.font(...)` on Images as well to make them scale with a dynamic text size and to make them, e.g., the same size as the text or proportionally larger.